src/modules/addScore.js

```diff
@@ -0,0 +1,13 @@
+export default async function addScore(Name, Score) {
+  await fetch('https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/FtbSsyyRO5eDxJO72kQu/scores/', {
+    method: 'POST',
+    headers: {
+      Accept: 'application/json',
+      'Content-Type': 'application/json',
+    },
+    body: JSON.stringify({
+      user: Name,
+      score: Score,
+    }),
+  });
```

Comment:
Please ensure to use arrow functions instead of the function keyword according to the project requirement.

Example of arrow function with async:

``
const addScore = async () => {
//TODO
}
``

![image](https://user-images.githubusercontent.com/10332499/156452663-03feb5ac-3622-4ba4-8667-bbf28ce98bd9.png)

Reply:
resolved 👍